samples/excel/85-preview-apis/worksheet-auto-filter.yaml

```diff
@@ -0,0 +1,164 @@
+order: 1
+id: excel-worksheet-auto-filter
+name: AutoFilter
+description: Adds an AutoFilter to a worksheet.
+host: EXCEL
+api_set:
+    ExcelApi: 1.9
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-auto-filter").click(() => tryCatch(addAutoFilter));
+        $("#randomize-data").click(() => tryCatch(randomizeData));
+        $("#refresh-auto-filter").click(() => tryCatch(refreshAutoFilter));
+        $("#remove-auto-filter").click(() => tryCatch(removeAutoFilter));
+
+        async function addAutoFilter() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getActiveWorksheet();
+                const farmData = sheet.names.getItem("FarmData").getRange();
```

Comment:
Is there a practical significance to using `getRange()` here instead of `getUsedRange()` ?